pmsoService/product_flow/features/orders/app/handlers.py

```
@@ -56,4 +63,47 @@ def change_status_sale_order(status_data: ChangeSaleOrderStatus, user: User):
+        'sale_orders': sale_orders,
+        'total_count': sale_orders.count(),
+        'selected_products': selected_products
+    }
+
+
+def generate_material_request(command: GenerateMaterialRequest, user: User):
+    sale_order_id = command.sale_order_id
+    
+    try:
+        sale_order = queries.get_by_id(sale_order_id)
+    except Exception:
```

Ruff (0.12.2)

Do not catch blind exception: `Exception`

(BLE001)